src/pages/delegates/delegates.ts

```diff
@@ -80,6 +81,11 @@ export class DelegatesPage implements OnDestroy {
   toggleSearchBar() {
     this.searchQuery.username = '';
     this.isSearch = !this.isSearch;
+    if (this.isSearch) {
+      setTimeout(() => {
+        this.searchbar.setFocus();
+      }, 100);
```

Comment:
why this timeout? 🤔

Reply:
since searchbar is not loaded instantly on pressing search button. If we dont set timeout, searchbar will not be defined

Reply:
Gotcha 👍